walkthrough/garden_cookiecutter/{{cookiecutter.directory_name}}/{{cookiecutter.short_name}}.meta.yml

```diff
@@ -1,19 +1,35 @@
+all_sources:
```

Comment:
I'm not sure about this. My current workflow is that I never modify the metadata in meadow, and only work on it in garden. On the one hand, it would be good to have all metadata as early as possible (from snapshot). On the other hand, what we have in snapshot should be as raw and close to the source as possible. So, if we want to add some custom descriptions, that should happen at a later stage (in garden). Also, while you are in garden, if you are inheriting the metadata straight from snapshot and want to make any amendments, you would need to re-run snapshot/meadow/garden, which is also a bit more inconvenient than just modifying the garden yaml file.
I'd suggest going with the current version and see how it goes.